README.md

```diff
@@ -1 +1,183 @@
-# cc-dashboard
\ No newline at end of file
+# Commanders Dashboard API
```

Comment:
Really nicely done here - the ReadMe is the first public-facing thing that anyone working with a repo sees.  It's always best practice to put some time and energy into making it look good.  Clearly, time was spent on that here to good effect!